src/codec.rs

```diff
@@ -280,6 +280,21 @@ impl<'a, T: ToOwned + ?Sized> Decode for ::std::borrow::Cow<'a, T> where
 	}
 }
 
+#[cfg(feature = "std")]
```

Comment:
Why std only?

Reply:
Just sticked with the convention that everything for `::std::` (like the two above) is featured gated behind that flag. I suppose in order to provide no-std/wasm compilation support (where those can't be needed anyways)!?!

Reply:
Yeah, I just wonder because it seems that using `PhantomData` in `no-std` environment should not harm and I also can imagine legit uses of it.